fabric_rti_mcp/eventstream/eventstream_service.py

```diff
@@ -0,0 +1,357 @@
+    """
+    endpoint = f"/workspaces/{workspace_id}/items/{item_id}"
+    
+    result = _run_async_operation(_execute_eventstream_operation("DELETE", endpoint))
+    return [result]
+
+
+def eventstream_update(
+    workspace_id: str,
+    item_id: str,
+    definition: Dict[str, Any]
+) -> List[Dict[str, Any]]:
+    """
+    Update an Eventstream item by workspace and item ID.
+    Authentication is handled transparently using Azure Identity.
+    
+    :param workspace_id: The workspace ID (UUID)
+    :param item_id: The eventstream item ID (UUID)
+    :param definition: Updated eventstream definition
+    :return: Updated eventstream details
+    """
+    # Prepare the eventstream definition as base64
+    definition_json = json.dumps(definition)
+    definition_b64 = base64.b64encode(definition_json.encode("utf-8")).decode("utf-8")
+    
+    payload: Dict[str, Any] = {
+        "definition": {
+            "parts": [
+                {
+                    "path": "eventstream.json",
+                    "payload": definition_b64,
+                    "payloadType": "InlineBase64"
+                }
+            ]
+        }
+    }
+    
+    endpoint = f"/workspaces/{workspace_id}/items/{item_id}"
+    
+    result = _run_async_operation(_execute_eventstream_operation("PUT", endpoint, payload))
+    return [result]
+
+
+def eventstream_get_definition(
+    workspace_id: str,
+    item_id: str
+) -> List[Dict[str, Any]]:
+    """
+    Get the definition of an Eventstream item.
+    Authentication is handled transparently using Azure Identity.
+    
+    :param workspace_id: The workspace ID (UUID)
+    :param item_id: The eventstream item ID (UUID)
+    :return: Eventstream definition
+    """
+    endpoint = f"/workspaces/{workspace_id}/items/{item_id}/getDefinition"
+    
+    result = _run_async_operation(_execute_eventstream_operation("POST", endpoint))
+    return [result]
+
+
+def _create_basic_eventstream_definition(
+    name: str, 
+    stream_id: Optional[str] = None
+) -> Dict[str, Any]:
+    """
+    Create a basic eventstream definition that can be extended later.
+    
+    :param name: Name for the default stream
+    :param stream_id: ID for the default stream (auto-generated if not provided)
+    :return: Basic eventstream definition
+    """
+    if stream_id is None:
+        stream_id = str(uuid.uuid4())
+        
+    return {
+        "compatibilityLevel": "1.0",
+        "sources": [],
+        "destinations": [],
+        "operators": [],
+        "streams": [
+            {
+                "id": stream_id,
+                "name": f"{name}-stream",
+                "type": "DefaultStream",
+                "properties": {},
+                "inputNodes": []
+            }
+        ]
+    }
+
+
+def eventstream_create_simple(
+    workspace_id: str,
+    name: str,
+    description: Optional[str] = None
+) -> List[Dict[str, Any]]:
+    """
+    Simple eventstream creation - just provide workspace and name.
```

Comment:
Why is this separate from the main tool? The description isn't super clear to me, I'm not sure I'd know where tool to call as a human.